runtime/queries/awk/highlights.scm

```diff
@@ -107,7 +107,7 @@
   "@load"
 ] @include
 
-"@namespace" @preproc
+"@module" @preproc
```

Comment:
unwanted change

Reply:
Yeah, I just realized I gotta watch out for those (luadoc, objc, and phpdoc are also tricky).